### PR TITLE
upload only images

### PR DESCRIPTION
**Pull request by victorpoliushko**

### CODE REVIEW

**Review by Dmitry528 — APPROVED** [review]
Good work

File: src/components/profile/add-place/add-place.js [hunk 1]
```diff
@@ -119,7 +119,8 @@ export default class AddPlace extends Component {
               value={this.state.filesValue}
               multiple
               required
-              onChange={this.fileSelectedHandler} />
+              onChange={this.fileSelectedHandler}
+              accept=".png, .jpeg, .jpg" />
```
Comment (marrybbb) [inline src/components/profile/add-place/add-place.js:123]: I did it on server previous sprint

image

Comment (Dmitry528) [reply src/components/profile/add-place/add-place.js:123]: In my opinion having a server-only validation is a bad idea, everything needs to be validated on the client side too

Comment (Dmitry528) [reply src/components/profile/add-place/add-place.js:123]: We don't need extra server requests

Comment (victorpoliushko) [reply src/components/profile/add-place/add-place.js:123]: user will not see any another files except theese. I think this is good, coz 2 factor validation can't be bad)

**Review by marrybbb — APPROVED** [review]

**Review by andrii-kondratiuk — APPROVED** [review]
Tested. Only .png, .jpeg, .jpg can be added. Good work.

**Review by Viktor-Hordynskyi — APPROVED** [review]

**Pull request merged**